docs/tutorials/exploring-datasets/movie-recommendation.md

```diff
@@ -185,28 +175,28 @@ This query has two parts:
 
 In the first part, we are looking for similar users. First, we need to define
 similar users: Two users are considered similar if they tend to give similar
-scores to the same movies. For the target user (Aladin) and some other user we
+ratings to the same movies. For the target user and some other user we
 are searching for the same movies:
 
 ```cypher
-MATCH (u:User {id:1000})-[r:Rating]-(m:Movie)-[other_r:Rating]-(other:User);
+MATCH (u:User {id:1000})-[r:RATED]-(m:Movie)-[other_r:RATED]-(other:User);
 ```
 
 But this is not enough for finding similar users. We need to choose users with
-the same movies and similar scores:
+the same movies and similar ratings:
 
 ```cypher
 WITH other.id AS other_id,
-     avg(abs(r.score-other_r.score)) AS similarity,
+     avg(abs(r.rating-other_r.rating)) AS similarity,
      count(*) AS similar_user_count
 WHERE similar_user_count > 2
 WITH other_id
 ORDER BY similarity
 LIMIT 10;
 ```
 
-Here we calculate similarities as the average distance between target user score
-and some other user score on the same set of movies. There are two parameters:
+Here we calculate similarities as the average distance between target user rating
+and some other user rating on the same set of movies. There are two parameters:
 similarUserCount limit (2) and similar user set size limit (10). Similar user
 count limit is used for filtering users who have at least 2 movies in common
 with the target user. Similar user set size is used to peek top 10 similar users
```

Comment:
This part is a bit weird with similarUserCount limit (2) and similar user set size limit (10) - maybe add code styling and explain better. I know this was here before, but since you're already fixing the tutorial 😄

Reply:
Updated this part about similar user count, it was a bit poor name, variable is for number of  same movies users have rated, so I renamed it to same_movies_rated.